content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/08-upgrade-from-mongodb-beta.mdx

```diff
@@ -22,7 +20,7 @@ We unfortunately can't cover all possible scenarios or changes required, but thi
 ## Requirements
 
 - Must be running MongoDB 4.2+ as a replica set (MongoDB Atlas does this for you automatically)
-- Node.js 12.6.x or 14.x or 16.x
+- Node.js 14.17.x or 16.x
 - TypeScript 4.1.x
```

Comment:
Should these also say "or higher" or something like that?
Because to me it sounds like it could be interpreted as if only Node.js 14.17.x (but not 14.18+) and TypeScript 4.1.x (but not 4.2+) were supported.

Reply:
Yes, I agree. This section isn't "minimum requirements", so this implies that you must have one of the stated versions. I'll add "or later".

Reply:
I've updated this now

Reply:
(We should remove these versions here completely, and just link somewhere else. This will be forgotten sooner or later when we upgrade again.)